aws/lambda/python/sns_converter.py

```
@@ -5,36 +5,59 @@
         }
-    } 
-    response = client.publish(TopicArn=sns_topic_arn,MessageStructure='json',Message=json.dumps({'default': json.dumps(message_to_sns)}))
-    return response
+
+        response = client.publish(
+            TopicArn=sns_topic_arn,
+            MessageStructure='json',
+            Message=json.dumps({'default': json.dumps(message_to_sns)}),
+        )
+        print(f"Successfully published message: {response['MessageId']}")
+        return response
```

Ruff

Consider moving this statement to an `else` block

(TRY300)